app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
```

Comment:
You could indicate that arguments that have default value are optional in a type annotation using the following syntax:
```suggestion
            is_hungry: bool | None = True
```